### ISSUE TITLE
[Kommentoerung 1.5.3] A_23737 - Clientmodul - Übermittlung von zusätzlichen Header-Informationen

### ISSUE THREAD

**Issue by CLAM01** [gematik/api-kim#39]
A_23737 - Clientmodul - Übermittlung von zusätzlichen Header-Informationen
Das Clientmodul MUSS beim Versand einer Nachricht die folgenden X-KIM Header erzeugen.

X-KIM-ToData 
Enthält Daten aus dem VZD-Eintrag des Empfängers (MAIL TO). 
Wenn mehrere professionOID oder specialization Attribute vorhanden sind, werden sie durch "|" getrennt. 
Format: {<MAIL RCPT TO>,<postalCode>,<professionOID>,<specialization>} 

X-KIM-CcData
Enthält die Daten aus dem VZD-Eintrag des Empfängers (MAIL CC). 
Wenn mehrere professionOID oder specialization Attribute vorhanden sind, werden sie durch "|" getrennt. 
Format: {<MAIL RCPT CC>,<postalCode>,<professionOID>,<specialization>}

**Frage:**
Wie soll vorgegangen werden, wenn es in einer KIM-Nachricht mehrere CC oder TO gibt.
Sollen dann n Header der gleichen Keynames vorhanden sein?
Es wird ja nur beschrieben dass prtofessionOID und specialization durch | getrennt werden müssen.

**Comment by gem-cp** [comment]
Eine Abstimmung mit dem BfDI erfolgt noch.

**Comment by dhufnagel** [comment]
Ich hatte es so gelesen, dass in dem Fall mehrerer TO oder CC Adressen, die Header pro Adresse gesetzt werden müssen. Ich halte die komplette AFO jedoch für problematisch, siehe #29

**Comment by CLAM01** [comment]
Ja, so hat die Afo den Anschein, was ich als problematisch sehe.
Das würde aber bei einer Mail an viele TO und CC einen enormen Overhead in den Headern erzeugen.
BTW. All diese Daten gehen danach zu lasten des LE, denn er kann dann immer weniger Daten versenden, da die Header Datenplatz belegen wenn es viele werden.

**Comment by dhufnagel** [comment]
Aus meiner Sicht ist es der falsche Weg, statistische Informationen zu erfassen, da die Komplexität des CmM erhöht wird, welches tausendfach clientseitig installiert wird. Die Daten kann man auch beim erheben der Statistik vom vzd abfragen.

**Comment by CLAM01** [comment]
Ok, die eigentliche Frage hat sich geklärt, nachdem man unter die Tabelle schaut und da den Satz findet: 

...Wenn mehrere Empfänger adressiert wurden, MUSS das jeweilige Header-Element mehrfach angegeben werden.[<=]
